hack/latest-kind-images.sh

```diff
@@ -61,8 +58,14 @@ LATEST_121_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_121_TAG)
 LATEST_122_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_122_TAG)
 LATEST_123_DIGEST=$(crane digest $KIND_IMAGE_REPO:$LATEST_123_TAG)
 
-cat << EOF
-# Copyright 2021 The cert-manager Authors.
+# v1.24 is a special hardcoded case for now, pending an official upstream 1.24 image
+
+TEMP_124_IMAGE_REPO=eu.gcr.io/jetstack-build-infra-images/kind
+LATEST_124_TAG=v1.24.0
+LATEST_124_DIGEST=sha256:2f170bf60cfad9d961711f96c34349d789a56b5783c9a5dbc0a29cb5a25ec729
+
+cat << EOF | tee ./devel/cluster/kind_cluster_node_versions.sh > ./make/kind_images.sh
```

Comment:
note: we used to just output to STDOUT in this script and we'd redirect it in our old makefile; seems easier to just write both files here.